README.md

```diff
@@ -56,3 +56,30 @@ b.gtld-servers.net.	172800	IN	AAAA	2001:503:231d::2:30
 END RequestId: b7abb31c-10a2-11e9-859c-b9daa8e62050
 REPORT RequestId: b7abb31c-10a2-11e9-859c-b9daa8e62050	Duration: 1206.05 ms	Billed Duration: 1300 ms 	Memory Size: 128 MB	Max Memory Used: 35 MB	
 ```
+
+---
+
+## Building a new version
+
+Please see the (AWS guide to create a deployment package](https://aws.amazon.com/premiumsupport/knowledge-center/lambda-linux-binary-package/). The following steps are tailored to `dig`.
+
```

Comment:
Might be better to link to this which has the exact AMI you should use to build Lambda layers: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html :)

Reply:
No worries, I've added that